pipeline/metadata/4.3.yaml

```diff
@@ -0,0 +1,534 @@
+  suite: "suites/nautilus/upgrades/tier-1_upgrade_container.yaml"
+  global-conf: "conf/nautilus/upgrades/tier-1_upgrade.yaml"
+  platform: "rhel-8"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - upgrades
+    - stage-1
+- name: "test-rbd-mirror-functionality"
+  suite: "suites/nautilus/rbd/tier-1_rbd_mirror.yaml"
+  global-conf: "conf/nautilus/rbd/tier-1_rbd_mirror.yaml"
+  platform: "rhel-8"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - rbd
+    - stage-2
+- name: "test-rgw-dynamic-resharding-greenfield"
+  suite: "suites/nautilus/rgw/tier-1_rgw_test-dynamic-resharding-greenfield.yaml"
+  global-conf: "conf/nautilus/rgw/tier-1_rgw_multisite.yaml"
+  platform: "rhel-8"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - rgw
+    - stage-2
+- name: "test-rgw-manual-resharding-greenfield"
+  suite: "suites/nautilus/rgw/tier-1_rgw_test-manual-resharding-greenfield.yaml"
+  global-conf: "conf/nautilus/rgw/tier-1_rgw_multisite.yaml"
+  platform: "rhel-8"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - rgw
+    - stage-2
+- name: "test-rgw-single-site"
+  suite: "suites/nautilus/rgw/tier-1_rgw.yaml"
+  global-conf: "conf/nautilus/rgw/tier-1_rgw.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7.9-server-x86_64.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7.9-minimal-amd64-3.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - rgw
+    - stage-2
+- name: "test-rgw-ssl-image"
+  suite: "suites/nautilus/rgw/tier-1_rgw_ssl_containerized-regression.yaml"
+  global-conf: "conf/nautilus/rgw/tier-1_rgw.yaml"
+  platform: "rhel-8"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-8-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-8-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - rgw
+    - stage-2
+- name: "test-upgrade-of-rpm-based-deployments"
+  suite: "suites/nautilus/upgrades/tier-1_upgrade_rpm.yaml"
+  global-conf: "conf/nautilus/upgrades/tier-1_upgrade.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7.9-server-x86_64.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7.9-minimal-amd64-3.yaml"
+  metadata:
+    - perbuild
+    - tier-1
+    - openstack
+    - ibmc
+    - upgrades
```

Comment:
Upgrades is not an FG in itself. So this test suite can contain dmfg also as another tag along with upgrades.

Reply:
Done added as
- upgrades
- dmfg